tests/Unit/Domain/Creators/TimeDependentOptions/Test_BinaryCompactObject.cpp

```diff
@@ -290,129 +294,127 @@ void test(const bool include_expansion, const bool include_rotation,
-                             &domain_outer_radius, &cube_a_center,
-                             &cube_b_center]() {
-      time_dep_options.build_maps(centers, cube_a_center, cube_b_center,
-                                  inner_outer_radii_A, inner_outer_radii_B,
-                                  domain_envelope_radius, domain_outer_radius);
-    };
-
-    // If we have excision info, but didn't specify shape map options, we
-    // should hit an error.
-    if (is_excised(excise_A) and not include_shape_a) {
-      CHECK_THROWS_WITH(build_maps(),
-                        Catch::Matchers::ContainsSubstring(
-                            "Trying to build the shape map for object"));
-      continue;
-    }
-    // If we have shape map options, but didn't specify excisions, we should
-    // hit an error.
-    if (include_shape_a and not is_excised(excise_A)) {
-      CHECK_THROWS_WITH(build_maps(),
-                        Catch::Matchers::ContainsSubstring(
-                            "No excision was specified for object"));
-      continue;
-    }
-
-    // Same for B
-    if (is_excised(excise_B) and not include_shape_b) {
-      CHECK_THROWS_WITH(build_maps(),
-                        Catch::Matchers::ContainsSubstring(
-                            "Trying to build the shape map for object"));
-      continue;
-    }
-    if (include_shape_b and not is_excised(excise_B)) {
-      CHECK_THROWS_WITH(build_maps(),
-                        Catch::Matchers::ContainsSubstring(
-                            "No excision was specified for object"));
-      continue;
-    }
+      const auto build_maps = [&time_dep_options, &centers,
+                               &inner_outer_radii_A, &inner_outer_radii_B,
+                               &domain_envelope_radius, &domain_outer_radius,
+                               &cube_a_center, &cube_b_center,
+                               filled_A = not excise_A,
+                               filled_B = not excise_B]() {
+        time_dep_options.build_maps(centers, cube_a_center, cube_b_center,
+                                    inner_outer_radii_A, inner_outer_radii_B,
+                                    filled_A, filled_B, domain_envelope_radius,
+                                    domain_outer_radius);
+      };
+
+      if constexpr (not IsCylindrical) {
+        if (include_shape_a and
+            (not excise_A and not transition_ends_at_cube_A)) {
+          CHECK_THROWS_WITH(build_maps(),
+                            Catch::Matchers::ContainsSubstring(
+                                "If the object is filled, the transition must "
+                                "end at the cube."));
+          continue;
+        }
+        if (include_shape_a and not excise_A) {
+          CHECK_THROWS_WITH(
+              build_maps(),
+              Catch::Matchers::ContainsSubstring(
+                  "the centers of the inner and outer object are different"));
```

Comment:
Where is this error from? I wasn't able to find it in the code base or this PR

Reply:
`src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/Wedge.cpp:75`

Reply:
Ugh that line break messed up my search.

Should this be caught one level up in the TimeDepBCO? Seems a bit odd to be getting an error from the wedge transition here

Reply:
Dunno, the logic in these tests is pretty complicated and took a while to work out because there are so many different configurations of the BCO domain so I don't feel particularly motivated to move things around